sdk/tests/deqp/functional/gles3/es3fLifetimeTests.js

```diff
@@ -400,20 +400,47 @@ es3fLifetimeTests.TfDeleteActiveTest = function(name, description) {
 setParentClass(es3fLifetimeTests.TfDeleteActiveTest, tcuTestCase.DeqpTest);
 
 es3fLifetimeTests.TfDeleteActiveTest.prototype.iterate = function() {
+/** @const */ var s_xfbVertexSource =
+    '#version 300 es\n' +
+    'void main ()\n' +
+    '{\n' +
+    ' gl_Position = vec4(float(1) / 2.0, float(1 % 2) / 2.0, 0.0, 1.0);\n' +
+    '}\n';
+
+/** @const */  var s_xfbFragmentSource =
+    '#version 300 es\n' +
+    'layout(location=0) out mediump vec4 dEQP_FragColor;\n' +
+    'void main (void)\n' +
+    '{\n' +
+    ' dEQP_FragColor = vec4(1.0, 1.0, 0.0, 1.0);\n' +
+    '}\n';
+    var buf = gl.createBuffer();
+
+    var sources = new gluShaderProgram.ProgramSources();
+    sources.add(new gluShaderProgram.VertexSource(s_xfbVertexSource));
+    sources.add(new gluShaderProgram.FragmentSource(s_xfbFragmentSource));
+    sources.add(new gluShaderProgram.TransformFeedbackMode(gl.SEPARATE_ATTRIBS));
+    sources.add(new gluShaderProgram.TransformFeedbackVarying('gl_Position'));
+    var program = new gluShaderProgram.ShaderProgram(gl, sources);
+    if (!program.isOk()) {
+        bufferedLogToConsole(program.getProgramInfo().infoLog);
+        testFailedOptions('failed to build program', true);
+    }
+    gl.useProgram(program.getProgram());
+
     var tf = gl.createTransformFeedback();
     gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, tf);
+    gl.bindBufferBase(gl.TRANSFORM_FEEDBACK_BUFFER, 0, buf);
+    gl.bufferData(gl.TRANSFORM_FEEDBACK_BUFFER, 48, gl.STATIC_DRAW);
+
     gl.beginTransformFeedback(gl.TRIANGLES);
     var errCode = gl.NONE;
-    try {
-        gl.deleteTransformFeedback(tf);
-    } catch (err) {
-        errCode = err.error;
-    }
-    gl.endTransformFeedback();
     gl.deleteTransformFeedback(tf);
+    errCode = gl.getError();
     assertMsgOptions(errCode == gl.INVALID_OPERATION,
         'Deleting active transform feedback must produce INVALID_OPERATION', false, true);
```

Comment:
Chrome will hang when running this test even with https://codereview.chromium.org/1752703003/. You may see a lot of gl errors, such as
[1291:1291:0301/225346:ERROR:gles2_cmd_decoder.cc(2101)] [.CommandBufferContext.RenderWorker-0x24b0aba1b6e0]GL ERROR :GL_INVALID_OPERATION : GLES2DecoderImpl::D
 error from previous GL command.

But after you remove above 4 lines, chrome will not hang. Similar thing can be seen for link_program case in negativeshaderapi.html. Also chrome will hang when you debug conformance2/transform_feedback/transform_feedback.html with inspector. Chrome doesn't reponse after beginTransformFeedback and drawArrays.  I guess there are still bugs in chrome for transform feedback.